packages/ui/src/services/ColorThemePlugin.ts

```diff
@@ -24,6 +24,40 @@ const defaultOptions = Vue.observable({
   },
 })
 
+export const THEME_NAMES = {
```

Comment:
Theme name holds no meaning as a enum. We don't keep any behavior for specific theme. Rather it's just a number of theme options.

So, for config:
```ts
export const colorThemes = {
  corporate: {primary: // ...},
  default: {primary: // ...}
}
```
would lessen complexity, while making a lot of sense.